tests/constants/esxcli/cmd.go

```diff
@@ -17,7 +17,8 @@
 package esxcli
```

Comment:
General comment: As we have decided to merge ... I would prefer something like `vsphereutil` certainly not `esxcli`